applications/salk-portal/src/components/ExperimentSidebar.jsx

```diff
@@ -45,6 +45,27 @@ const useStyles = makeStyles({
       },
     },
 
+    '& .population-label-container': {
+      display:'flex',
+      flexDirection:'row',
+      alignContent:'center',
```

Comment:
@paularah if you use alignitems it would align the text with the center of the switch as in figma